crates/win-api-wrappers/src/rpc.rs

```diff
@@ -116,6 +116,7 @@ pub struct RpcServerInterfacePointer {
     pub raw: &'static RPC_SERVER_INTERFACE,
 }
 
+// SAFETY: RpcServerInterfacePointer could be sent between threads.
 unsafe impl Send for RpcServerInterfacePointer {}
```

Comment:
issue: This is not explaining why. Implementing `Send` on `RpcServerInterfacePointer` _means_ that `RpcServerInterfacePointer` can be sent to another thread. Why is it safe to do so is the purpose of `SAFETY` comments. For this to be `Send`, we must not share mutable state with something else without enforcing exclusive access to it. This depends on `RPC_SERVER_INTERFACE` and the operations called on it by our wrapper. See https://doc.rust-lang.org/nomicon/send-and-sync.html